src/main/resources/hudson/security/table.js

```diff
@@ -40,15 +40,15 @@ Behaviour.specify(".matrix-auth-add-button", 'GlobalMatrixAuthorizationStrategy'
 
     var tooltipAttributeName = getTooltipAttributeName();
 
-    findElementsBySelector(copy, ".stop a").each(function(item) {
+    findElementsBySelector(copy, ".stop a").forEach(function(item) {
       let oldTitle = item.getAttribute("title");
       if (oldTitle !== null) {
         item.setAttribute("title", oldTitle.replace("__SID__", name).replace("__TYPE__", typeLabel));
       }
       item.setAttribute(tooltipAttributeName, item.getAttribute(tooltipAttributeName).replace("__SID__", name).replace("__TYPE__", typeLabel));
     });
 
-    findElementsBySelector(copy, "input[type=checkbox]").each(function(item) {
+    findElementsBySelector(copy, "input[type=checkbox]").forEach(function(item) {
```

Comment:
Tooltips for checkboxes in newly added rows don't have placeholders, but proper labels.